src/lib/components/key-value/KeyValue.svelte

```diff
@@ -301,61 +342,137 @@
                   />
                 {/if}
               </div>
-              <div class="flex-grow-1 w-100 position-relative">
-                <input
-                  type="text"
-                  placeholder="Enter Value"
-                  class="form-control keyValuePair py-1"
-                  style="font-size: 13px;"
-                  id={"pair-value" + index}
-                  disabled={mode == "READ" ? true : false}
-                  bind:value={element.value}
-                  on:input={() => {
-                    updateParam(index);
-                    tempText = element.value;
-                    handleInputValue();
-                  }}
-                  on:keyup={(e) => {
-                    trackCursor = e.target.selectionStart;
-                  }}
-                  on:blur={() => {
-                    setTimeout(() => {
-                      tempText = "";
-                      trackParanthesis = [];
-                      trackCursor = undefined;
-                      filterData = [];
-                    }, 300);
-                  }}
-                  on:focus={(e) => {
-                    tempText = element.value;
-                    focusedInput = index;
-                    focusedElement = "value";
-                    handleInputValue();
-                    const elem = document.getElementById("pair-value" + index);
-                    environmentAxisY = elem.getBoundingClientRect().top + 30;
-                    environmentAxisX = elem.getBoundingClientRect().left;
-                  }}
-                />
-                {#if focusedInput === index && focusedElement === "value" && trackParanthesis.length === 2 && filterData.length > 0}
-                  <EnvironmentPicker
-                    {environmentAxisX}
-                    {environmentAxisY}
-                    {filterData}
-                    inputText={element.value}
-                    {trackCursor}
-                    {trackParanthesis}
-                    updateText={(url) => {
-                      element.value = url;
-                    }}
-                    handleInputValue={() => {
+              {#if type === "file"}
+                <div class="flex-grow-1 w-100">
+                  <div
+                    class="position-relative rounded p-1 d-flex backgroundColor"
+                    style="height: 27px;"
```

Comment:
height 27px fixed any reason?

Reply:
Actually we have given this as per Figma, so the height of input text for file type for key value component stays the same wherever we have used this, for text type it is also the same.